scripts/workspace-conversion/README.md

```diff
@@ -0,0 +1,61 @@
+## Converting Workspace Dataset Attributes to Catalog Entry Data
+
+# Description
+This script takes the workspace dataset attributes and attempts to map them to similar fields in the data catalog service as catalog entries.
+
+# Dataset Attributes
+An example workspace with attributes can be found [here](https://github.com/DataBiosphere/terra-ui/blob/dev/src/data/workspace-attributes.js#2), under the variable `displayLibraryAttributes`.
+
+The request made to retrieve the workspace details is an XHR request to rawls to retrieve the workspace information. The basic structure of the return of the call looks like:
+
+```
+{
+  owners: [],
+  workspaces: {
+    attributes: {
+      description: string,
+      `library:cellType: string,
+      `library: ...
+    }
+  }
+}
+```
+
+What is shown to the user in that table is the contents in the attributes that terra-ui has determined is important to display. These values are hardcoded at 
+`{terra-ui}/src/data/workspace-cleanAttributes.js.displayLibraryAttributes`
+
+# Value mappings
+To run the conversion from workspace to dataset, here is a current mapping scheme:
+
+
+| i | Attribute | Rawls Workspace Path | Data Catalog Entry Path |
+| - | --------- | -------------------- | -------------------------- |
+| 1 | Cohort Phenotype/Indication (Disease Ontology)  | `library:diseaseOntologyLabel` | `samples.disease.0`<br>kathy: lists predefined terms, but forces them to stick with a specific vocabulary. The other phenotype field is the diseases |
+| 2 | No. of Subjects | `library:numSubjects` | `counts.donors`<br>Kathy: suggests leaving out this field because it is volatile and can easily become inaccurate |
+| 3 | Data Category | `library:dataCategory`<br>ex: "Simple Nucleotide Variation, Copy Number Variation, Expression Quantification, DNA-Methylation, Clinical phenotypes, Biosample metadata" | `dct:dataCategory`*<br>This field does not currently exist |
```

Comment:
Reviewing these examples it's pretty much a duplication of what's in Experimental Strategy (that we'll map to data modality).  I'm not sure if we need it at all but if there's concern about losing the information, please feel free to add this as a column. It won't be included in the model.

Reply:
Ok! We can add it as a column / new field, but with no expectation that anyone else will see or use it :)